src/styles/blocks/_catalog.scss

```diff
@@ -14,9 +14,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: $time;
```

Comment:
specify the properties for which you'll set transition

Reply:
ok